airflow/www/static/js/dag/StatusBox.tsx

```diff
@@ -50,8 +53,13 @@ export const StatusWithNotes = ({
       background={getStatusBackgroundColor(color, !!containsNotes)}
       borderRadius="2px"
       borderWidth={state ? 0 : 1}
+      display="flex"
+      alignItems="center"
+      justifyContent="center"
       {...rest}
-    />
+    >
+      {!!tryNumber && tryNumber > 1 && <MdRefresh color="white" />}
```

Comment:
Usually I tend to not do `!!` operator on numbers, because usually people to that to check if the variable is defined, while it won't work for `0`. `!!0 = false`.  Here you really want to check if it is defined so an explicit `tryNumber !== undefined` if better IMHO.

People copy pasting that code without the specific case here `tryNumber > 1`  will have surprises.

Reply:
I've updated the code 👍